pycbc/io/hdf.py

```diff
@@ -648,7 +648,8 @@ def get_column(self, cname):
         # If the mask accesses few enough elements then directly use it
         # This can be slower than reading in all the elements if most of them
         # will be read.
-        if self.mask is not None and (self.mask.sum() < len(self.mask) * MFRAC):
+        if self.mask is not None and (isinstance(self.mask, list) or \
+                (self.mask.sum() < len(self.mask) * MFRAC)):
```

Comment:
Does this cover all possible cases?  Could self.mask in practice be a list of indices *or* an array of indices *or* a list of Booleans *or* (the default assumption?) an array of Booleans?

Reply:
I believe all of these are possible, and this covers all cases. I don't think we use lists of Booleans in practice (arrays of Booleans are used a lot), and that case might be a little sub-optimal.

We could optimize the "array of indices" case more ... Actually I'll make that change

Reply:
Done